packages/bpk-component-infinite-scroll/src/withInfiniteScroll.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
Sorry I din't catch it early but we need flow annotation here too. You should also export the flow props definition here and in `index.js'. ` Here's an example: https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-navigation-bar/src/BpkNavigationBar.js#L30

Reply:
added flow annotations. 😓